### DISCUSSION TITLE
Yes/No/Abstain Ballot with Maximum amount of votes

### DISCUSSION THREAD

**Discussion by ManniLoe** [OpenSlides/OpenSlides#5984] [Q&A]
Hi,
Why can't a maximum amount of votes be used in a YNA election as in a "simple" yes election?

We have the case that in an election with Yes / No / Abstain per candidate only as many "yes votes" may be given as there are people to be elected.

Is there a way to patch this?

Thx, Best regards

**Comment by FinnStutzenstein** [comment]
Is minimum amounts currently possible?

**Reply by ManniLoe** [reply]
![image](https://user-images.githubusercontent.com/1992033/113250353-9ff5e300-92c0-11eb-81cf-55e6d1c6713f.png)
No, in the YNA poll i can't set minimum or maximum amount.

**Comment by tsiegleauq** [comment]
This is a feature request.

We could not come up with a usable way to support YNA in combination with min and max amount.
Both the UI/UX setup to support this many controls and the calculation of the results were unresolved or unsatisfactory, thus we decided not to support this at the moment.
